packages/node_modules/@ciscospark/widget-recents/src/components/RecentsHeader.css

```diff
@@ -1,14 +1,14 @@
 .recentsHeader {
-  height: 60px;
+  height: 70px;
```

Comment:
Web client header is 60px. Why the change?

Reply:
yes. 60px is fine.